worker/exec/konnector.go

```diff
@@ -370,7 +370,7 @@ func (w *konnectorWorker) ensurePermissions(inst *instance.Instance) error {
 	}
 	value := consts.Konnectors + "/" + w.slug
 	for _, rule := range perms.Permissions {
-		if rule.Type == consts.Files && rule.Selector == couchdb.SelectorReferencedBy {
+		if permission.MatchType(rule, consts.Files) && rule.Selector == couchdb.SelectorReferencedBy {
```

Comment:
I don't think we want to support wildcard permission with a referenced_by selector. I would have let this line of code like it was.